### DISCUSSION TITLE
🖐️ Monorepo support

### DISCUSSION THREAD

**Discussion by drmikecrowe** [dagger/dagger#12194] [Help]
### What happened? What did you expect to happen?

Does anybody have any examples of a monorepo?  For example, I'm trying to learn how to:

- Creating the buildEnv at the top level and leveraging this for all apps
- Caching of `node_modules` in each projects (some node tooling we use forces us to use individual node_modules at the moment)
- Supporting 2 source folders (`apps/` and `packages/` -- both of which are used)

**Comment by mircubed** [comment]
@vikram-dagger This is an issue to keep in mind for the documentation that you are working on. @TomChv can sync with you on the best practices.

**Comment by TomChv** [comment]
I'm removing the `sdk/typescript` label since it's not a specific TS issue and we should write guidelines for every SDKs :)
